.github/workflows/tidy.yml

```
@@ -94,21 +94,23 @@ jobs:
-      # Whole-tree formatters, after the parallel join so they don't race a regenerator.
-      - name: Format Markdown
-        if: steps.precheck.outputs.should-run == 'true'
-        run: ./target/debug/wado-dev-tools format-md
-
-      - name: Format Wado sources
-        if: steps.precheck.outputs.should-run == 'true'
-        run: ./target/debug/wado format -w .
+      # Whole-tree formatters (`mise run format` locally), separate steps so each
+      # is timed and fails on its own. After the join so they don't race a
+      # regenerator; these two take no cargo lock.
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)